src/components/pipelines-metrics/PipelinesAverageDuration.tsx

```diff
@@ -36,6 +38,40 @@ interface PipelinesAverageDurationProps {
 }
 type DomainType = { x?: DomainTuple; y?: DomainTuple };
 
+const getChartData = (
+  tickValues: number[] | Date[],
+  data: SummaryResponse,
+  type: string,
+) => {
+  const chartData = tickValues?.map((value) => {
+    const s = data?.summary.find((d) => {
```

Comment:
initially data?.summary might be undefined and will throw a typeError
```suggestion
    const s = data?.summary?.find((d) => {
```